lib/mastodon/cli/maintenance.rb

```diff
@@ -534,7 +534,7 @@ def deduplicate_tags!
 
       say 'Deduplicating tags…'
       ActiveRecord::Base.connection.select_all("SELECT string_agg(id::text, ',') AS ids FROM tags GROUP BY lower((name)::text) HAVING count(*) > 1").each do |row|
-        tags = Tag.where(id: row['ids'].split(',')).sort_by { |t| [t.usable?, t.trendable?, t.listable?].count(false) }
+        tags = Tag.where(id: row['ids'].split(',')).order(:usable, :trendable, :listable)
```

Comment:
I would keep the existing code.

Reply:
Updated to preserve tag order from previous.